pkg/trace/config/apply.go

```diff
@@ -405,3 +408,32 @@ func splitString(s string, sep rune) ([]string, error) {
 
 	return r.Read()
 }
+
+func toFloat64(val interface{}) (float64, error) {
+	switch v := val.(type) {
+	case float64:
+		return v, nil
+	case float32:
```

Comment:
nit: We can put all of these numbers on the same line: `case float32, int, int32, int64, etc:`

Reply:
Tried that too. It doesn't work. `v` ends up being of type `interface{}`